metrics/cli/README.md

```diff
@@ -1,34 +1,69 @@
-# Metrics deploy CLI
+# Metrics CLI
```

Comment:
What do you think about adding a bit more info about how does the Metrics CLI works? For example, mention that this tool downloads the source code by itself and it is not required to check out the code from the repository before running the Metrics CLI. Also, I believe it would be useful to mention that the Metrics CLI does not store any data entered during the deployment process. Overall, I believe that the [Metrics readme](https://github.com/platform-platform/monorepo/tree/master/metrics) and [Monorepo readme](https://github.com/platform-platform/monorepo) are great references to use during this readme creation.